### PR TITLE
Additional getPreferredVersion updates to overview

### PR DESCRIPTION
**Pull request by benjaminapetersen**

### CODE REVIEW

File: app/scripts/controllers/overview.js [hunk 1]
```diff
@@ -80,11 +80,14 @@ function OverviewController($scope,
...
 
-  // API versions
+  var deploymentsVersion = APIService.getPreferredVersion('deployments');
+  var horizontalPodAutoscalersVersion = APIService.getPreferredVersion('horizontalpodautoscalers');
   var serviceBindingsVersion = APIService.getPreferredVersion('servicebindings');
   var serviceClassesVersion = APIService.getPreferredVersion('clusterserviceclasses');
   var serviceInstancesVersion = APIService.getPreferredVersion('serviceinstances');
   var servicePlansVersion = APIService.getPreferredVersion('clusterserviceplans');
+  var statefulSetsVersion = APIService.getPreferredVersion('statefulsets');
+  var replicalSetsVersion = APIService.getPreferredVersion('replicasets');
```
Comment (jwforres) [inline app/scripts/controllers/overview.js]: typo here and below, replicaSetsVersion

Comment (benjaminapetersen) [reply app/scripts/controllers/overview.js]: +1 got it

**Comment by benjaminapetersen** [comment]
thx, updated

File: app/scripts/controllers/overview.js [hunk 2]
```diff
@@ -80,11 +80,16 @@ function OverviewController($scope,
...
-  // API versions
+  var deploymentsVersion = APIService.getPreferredVersion('deployments');
+  var horizontalPodAutoscalersVersion = APIService.getPreferredVersion('horizontalpodautoscalers');
   var serviceBindingsVersion = APIService.getPreferredVersion('servicebindings');
   var serviceClassesVersion = APIService.getPreferredVersion('clusterserviceclasses');
   var serviceInstancesVersion = APIService.getPreferredVersion('serviceinstances');
   var servicePlansVersion = APIService.getPreferredVersion('clusterserviceplans');
+  var statefulSetsVersion = APIService.getPreferredVersion('statefulsets');
+  var replicaSetsVersion = APIService.getPreferredVersion('replicasets');
+  $scope.buildConfigsInstantiateVersion = APIService.getPreferredVersion('buildconfigs/instantiate');
```
Comment (spadgett) [inline app/scripts/controllers/overview.js]: This is a component, so we shouldn't be adding it to scope directly.

```
overview. buildConfigsInstantiateVersion = ...
```

Comment (benjaminapetersen) [reply app/scripts/controllers/overview.js]: Ah, thats right, we have that pattern for overview.  updating, just a min.

Comment (spadgett) [reply app/scripts/controllers/overview.js]: Yeah it uses controllerAs

https://github.com/openshift/origin-web-console/blob/master/app/scripts/app.js#L73-L78

**Comment by benjaminapetersen** [comment]
updated

**Comment by spadgett** [comment]
Thanks @benjaminapetersen 

/lgtm

**Pull request merged**